sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/CheckpointStore/BlobsCheckpointStoreTests.cs

```diff
@@ -621,7 +621,7 @@ public async Task ListCheckpointsUsesSequenceNumberAsTheStartingPositionWhenNoOf
         /// </summary>
         ///
         [Test]
-        public async Task ListCheckpointsUsesSequenceNumberAsTheStartingPositionWhenOffsetIsNullInLegacyCheckpoint()
+        public async Task ListCheckpointSkipsCheckpointsWhenOffsetIsNullInLegacyCheckpoint()
```

Comment:
Is it never possible to have a valid sequence with a null offset?   If so, we may want to have a couple of additional test scenarios.

Reply:
T1 was skipping all checkpoints with empty/null offset https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/AzureStorageCheckpointLeaseManager.cs#L125

I might need a test for empty.